sklearn/compose/_column_transformer.py

```diff
@@ -540,6 +543,7 @@ def fit_transform(self, X, y=None):
 
         self._update_fitted_transformers(transformers)
         self._validate_output(Xs)
+        self._output_dims = [X.shape[1] for X in Xs]
```

Comment:
Is this used?

Reply:
Thanks for spotting this. It was used in the original PR for a feature which was since removed. I'll update it now